doc/Install.md

```diff
@@ -145,6 +145,16 @@ Example:
 
 *Note*: Do **not** rename the .htaccess-dist file as it is tracked by GIT and renaming will cause a dirty working directory.
 
+### Verify the "host-meta" page is working
+
+Friendica should respond automatically to an important address named /.well-known/host-meta  This URL would look like, for example, https://example.com/.well-known/host-meta  It must be visible to the public and must respond with an XML file that is automatically customized to your site.
+
+If that URL is not working, it is possible that some other software is using the /.well-known/ path and is not configured correctly to work with Friendica.  Other symptoms may include an error message in the Admin settings that says "host-meta is not reachable on your system. This is a severe configuration issue that prevents communication."  Another common error related to host-meta is the, "Invalid profile URL."
+
+Check for a .well-known directory that did not come with Friendica.  Any /.well-known/.htaccess file could interfere with this core requirement.  You should remove any RewriteRules from that file, or remove that whole file if appropriate.
```

Comment:
To be precise: There mustn't be any real ```.well-known``` path in the installation. It is only create via the rewrite rules.

Reply:
The existence of /.well-known/ does not prevent re-writing of /.well-known/host-meta as it will not pass the -f check for host-meta.  I do agree that if /.well-known/host-meta exists as a file it will cause a huge problem.

Reply:
It's the best to completely remove that folder if it exists. It doesn't serve any purpose.

Reply:
The /.well-known directory is used by LetsEncrypt.  It's really not possible to remove it in this situation.

Reply:
I'm using LetsEncrypt - and doesn't have this folder.

Reply:
I use two different LetsEncrypt clients on two different servers and both use file-based HTTP validation inside the /.well-known directory.  I realize there are other clients, but you risk alienating LetsEncrypt users by telling them they can't do this.

Reply:
You could describe it this way: "Normally there shouldn't be any '.well-known' path in your Friendica installation. But if you need one, e.g. for some alternate letsencrypt client, then assure that ..."